.flake8

```diff
@@ -7,6 +7,5 @@ exclude =
     geoapi/schemas/__init__.py
     # Files below are tied with https://jira.tacc.utexas.edu/browse/DES-2267
     geoapi/initdb.py
-    geoapi/tasks/streetview.py  
```

Comment:
We have a task (DES-2267) to clean up some files that have linting errors. i used this as an opportunity to clean up one of the files.